src/utils.js

```diff
@@ -101,6 +101,12 @@ const isWithinLastWeek = date => {
   return daysDifference >= 0
 }
 
+const isBeforeToday = date => {
+  if (isToday(date)) return false
```

Comment:
since `isAfterToday` exists already would it be better to just do
```
const isBeforeToday = date => !(isToday(date) || isAfterToday(date))
```

Reply:
Good idea, cheers